suse/rdma-core.spec

```diff
@@ -330,17 +328,17 @@ rdma-ndd is a system daemon which watches for rdma device changes and/or
 hostname changes and updates the Node Description of the rdma devices based
 on those changes.
 
-%if %{with_pyverbs}
 %package -n python3-pyverbs
 Summary:        Python3 API over IB verbs
 Group:          Development/Languages/Python
 
 %description -n python3-pyverbs
 Pyverbs is a Cython-based Python API over libibverbs, providing an
 easy, object-oriented access to IB verbs.
-%endif
 
 %prep
+# Make sure LTO is disable as rdma-core fails to compile with LTO enabled
```

Comment:
What? Why? Really?

Reply:
See: https://bugzilla.suse.com/show_bug.cgi?id=1133099

Reply:
Hum. LTO is broken in GCC if combined with shared libraries that use symbol versioning:

https://stackoverflow.com/questions/46304742/how-to-combine-lto-with-symbol-versioning

Which means it is basically crazy for a distro to turn this on by default - you will quietly and randomly break  any shared libraries that use symbol versioning.

Reply:
They're slowly switching a lot of packages to LTO. Once the broken one have been identified and fixed they'll probably enable it distro wide:
https://lists.opensuse.org/opensuse-factory/2019-04/msg00320.html

Reply:
Well, we need a compiler fix, and rdma-core produces static libraries so we can't ever enable it via global build flags anyhow (cmake will have to finesse it and exclude the static libraries).. 

But OK.. If this is how suse is going.. We can revisit it in gcc 10

Reply:
SUSE does not produce static libraries. But yes we'll see when GCC10 gets here